### ISSUE TITLE
Clear search field icon in wrong position

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-admin#1217]
The positioning of the clear icon (x) for the search field is floating randomly. We should be able to fix it by extending the input length so it is a fixed position from the right end.

![image](https://user-images.githubusercontent.com/555033/119281377-a2a20280-bc89-11eb-9281-57ca39c3b965.png)

**Comment by michalkleiner** [comment]
@clarkepaul when focus is in the search field, I see a key hint there. When the input field is blurred, the hint disappears, creating the gap as in your screenshot.
<img width="238" alt="Screen Shot 2021-05-24 at 4 12 38 PM" src="https://user-images.githubusercontent.com/233342/119295510-17d1ff80-bcab-11eb-8e14-686abb47a6a9.png">

**Comment by clarkepaul** [comment]
@michalkleiner yup noticed that too after posting. Should be able to design or figure a way of accommodating both I hope. Don't have the answer right now, just the problem :)

**Comment by GuySartorelli** [comment]
The `x` isn't present anymore, and I don't think we need to bring it back so this isn't a problem anymore.
